public/stylesheets/main.css

```diff
@@ -418,6 +422,7 @@ a.carousel-control.right{
 
 #conditionPanel {
     display: flex;
+    overflow: scroll;
```

Comment:
I've pulled the code and I see horizontal and vertical scrolls in a condition panel (by default) even they are not needed.. will send a screenshot

Reply:
Can you please show me? I can't reproduce it here
Might be a linux thing. Scrollbars are handled by the OS